app/javascript/packages/document-capture/context/help-center.jsx

```diff
@@ -0,0 +1,56 @@
+import { createContext } from 'react';
+import { addSearchParams } from '@18f/identity-url';
+
+/** @typedef {import('react').ReactNode} ReactNode */
+
+/**
+ * @typedef HelpCenterURLParameters
+ *
+ * @prop {string} category
+ * @prop {string} article
+ * @prop {string} location
+ */
+
+/**
+ * @typedef {(params: HelpCenterURLParameters) => string} GetHelpCenterURL
+ */
+
+/**
+ * @typedef HelpCenterContext
+ *
+ * @prop {string} helpCenterRedirectURL
+ * @prop {GetHelpCenterURL} getHelpCenterURL
+ */
+
+const HelpCenterContext = createContext(
+  /** @type {HelpCenterContext} */ ({
+    helpCenterRedirectURL: '',
+    getHelpCenterURL: (params) => addSearchParams('', params),
+  }),
+);
+
+HelpCenterContext.displayName = 'HelpCenterContext';
+
+/**
+ * @typedef HelpCenterContextProviderProps
+ *
+ * @prop {Omit<HelpCenterContext, 'getHelpCenterURL'>} value
+ * @prop {ReactNode} children
+ */
+
+/**
+ * @param {HelpCenterContextProviderProps} props
+ */
+function HelpCenterContextProvider({ value, children }) {
+  /** @type {GetHelpCenterURL} */
+  const getHelpCenterURL = (params) => addSearchParams(value.helpCenterRedirectURL, params);
```

Comment:
to me this seems more like a `build` method than a `get` method?

Reply:
> to me this seems more like a `build` method than a `get` method?

Hm, could you explain more? Is the idea that a `get`-prefixed method would only return a value directly, not derive something? I guess I've not made much a distinction on this previously, or at least would be happy to just offer up the value directly if that's what a "get" would imply.

I don't feel too strongly, but more worry to the point of: We haven't made much of a distinction in the past, so an argument could be made to the point of "consistency for consistency's sake" 🤷

Reply:
in my head, "get" is like "retrieve" like a load from a database or a hash or something

whereas in this case, we're not retrieving anything, we're building a new string based on some immediate values

But this is just personal style, like you said, we don't have a codebase style so I really don't feel that strongly about enforcing it

Reply:
Okay, I think I'll plan to keep it as-is for now, though perhaps separately we can think about / document naming conventions around methods.